miasm2/analysis/dse.py

```diff
@@ -339,18 +346,20 @@ def callback(self, _):
             # Update the concrete execution
             self._update_state_from_concrete_symb(self.symb_concrete)
             while True:
+
                 next_addr_concrete = self.symb_concrete.run_block_at(cur_addr)
                 self.symb.run_block_at(cur_addr)
 
-                if not(expr_is_label(next_addr_concrete) and
-                       next_addr_concrete.name.offset is None):
-                    # Not a lbl_gen, exit
+                if not isinstance(next_addr_concrete, ExprLoc):
```

Comment:
The test was checking that `next_addr_concrete` points (or not) on a "generated block". I'm not really sure that the new test is equivalent

Reply:
Correct!
done